naming/src/main/java/com/alibaba/nacos/naming/core/v2/client/manager/ClientManagerDelegate.java

```diff
@@ -105,7 +103,7 @@ private ClientManager getClientManagerById(String clientId) {
         if (isConnectionBasedClient(clientId)) {
             return connectionBasedClientManager;
         }
-        return clientId.endsWith(SUFFIX) ? persistentIpPortClientManager : ephemeralIpPortClientManager;
+        return clientId.endsWith(IpPortBasedClient.PERSISTENT_SUFFIX) ? persistentIpPortClientManager : ephemeralIpPortClientManager;
```

Comment:
No need to move `SUFFIX` to `IpPortBasedClient`

Reply:
If we don't move `SUFFIX` to `IpPortBasedClient`, we should declare another variable named `SUFFIX` in `OperatorController` to judge whether `clientId` is persistent clientId.
So I think `PERSISTENT_SUFFIX` is a property of `IpPortBasedClient`.
Otherwise, we should change `SUFFIX` from private  to public, in order `OperatorController` can reference it.

Reply:
Can we move it to ClientConstants?

Reply:
And replace all `false` with same meaning usage.

Reply:
Move it to ClientConstants is a good choice.
It seems that persistent clientId judgement is only occurs in `ClientManagerDelegate` and `OperatorController` so far.

Reply:
Should we move `ID_DELIMITER` from `IpPortBasedClient` to `ClientConstants` too?